ipn/backend.go

```diff
@@ -46,12 +46,13 @@ func (s State) String() string {
 		"Running"}[s]
 }
 
-// EngineStatus contains WireGuard engine stats.
+// EngineStatus contains WireGuard engine metadata and stats.
 type EngineStatus struct {
 	RBytes, WBytes int64
 	NumLive        int
 	LiveDERPs      int // number of active DERP connections
 	LivePeers      map[key.NodePublic]ipnstate.PeerStatusLite
+	LocalAddrs     []tailcfg.Endpoint // the set of possible endpoints for the magic conn
```

Comment:
This seems like a layering violation. EngineStatus has always been just about wireguard and not magicsock.

But I guess it's kinda morphed over time.

In any case, this doc comment ("the set of possible endpoints for the magic conn") doesn't seem to match the name ("Local" in "LocalAddrs") ... you can have endpoints that aren't local.

I'd instead add a new ipn.Notify field type like "NewConfig" to broadcast new global config probaby.